### DISCUSSION TITLE
Is it possible to create multiple free plan accounts and organizations.

### DISCUSSION THREAD

**Discussion by yutsugim** [community/community#56683] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi.
I have two questions about accounts and organizations policy.

1. Is it possible to create multiple free plan account?
I am currently using my GitHub account for private purpose, in addition to its use for work at my company. The terms of service appear to prohibit one person from creating multiple free accounts. However, at present, confidential code from my company is accessible from my personal PC. In this situation, is it possible to hold a free account for each organization? If that is feasible, is it necessary for the organization to be on a paid plan?

2. Is it possible to create multiple free plan organizations?
Apologies for the rudimentary question, but I am curious if it's permissible to create several organizations under the free plan. As far as I understand from the term of service, there do not seem to be explicit limitations to this. However, I would like to confirm this explicitly for the sake of my company's compliance.

Thank you.

**Comment by Itz0xAkira** [comment]
Hello there :D,
  well for your first question no its not possible to have alt accounts its against the tos :( , "One person or legal entity may maintain no more than one free Account (if you choose to control a machine account as well, that's fine, but it can only be used for running a machine)." [https://help.github.com/en/github/site-policy/github-terms-of-service](url)
  
  but for your second question if it is under one account you can have multiple organizations.
hope this helped.
